lib/price-import.coffee

```diff
@@ -134,6 +137,15 @@ class PriceImport extends ProductImport
     debug 'About to send %s requests', _.size(posts)
     Promise.settle(posts)
 
+  ###
+   * returns true if product does not have any staged changes
+   * and is published
+  ###
+  _canBePublished: (product) ->
+    if !product.hasStagedChanges and product.published
```

Comment:
@panshul007 In my opinion this is not a repository for covering individual use cases but having a more or less generic solution for all. What do you think @mmoelli @emmenko @hajoeichler @PhilippSpo?